### PR TITLE
Styles/weather components

### PR DESCRIPTION
**Pull request by rachelpr**
-Changed the weather icons to svg files (keeping old icon folder around just in case something got borked)
-Added a bunch of styling to be like the wireframe
Screen Shot 2022-08-09 at 11 23 26 PM

Let me know yer thoughts! Def still a WIP but getting closer 🦅

### CODE REVIEW

**Comment by rachelpr** [comment]
I think the border on the component is black...which is not intentional, but will be changed!

File: leafsmart-front-end/src/components/Dashboard.js [hunk 1]
```diff
@@ -35,27 +35,27 @@ const Dashboard = () => {
...
+              </Link>
+              <Search onSearchChange={handleOnSearchChange} />
             </div>
             {cityName && <Events cityName={cityName} />}
           </div>
           {cityName && (
             <div className="ml-12">
-              <CurrentWeather coordinates={coordinates}/>
-              <ForecastWeather coordinates={coordinates} />
+              <div className=" border rounded-3xl p-4 mb-8 mt-4 w-486 bg-gradient-to-r from-Independence to-HeliotropeGray">
```
Comment (b1u3too) [inline leafsmart-front-end/src/components/Dashboard.js:47]: Looks beauty! I think the border class is the only thing betrayin' us here, but that's easy enough to remove! :D

**Review by b1u3too — APPROVED** [review]
Looks good! Only ask would be nixing that border before merging to main!

**Comment by rachelpr** [comment]
can do's!

**Review by MGS31 — APPROVED** [review]
Approved all looks good to me!

**Pull request merged**